### ISSUE TITLE
Timeout was reached

### ISSUE THREAD

**Issue by lhy2749** [triton-inference-server/server#5331]
**Description**
I need to use s3 and k8s to deploy the model. It may be the internal bandwidth problem of k8s. I am sure that I can access the s3 service, but I cannot download a larger model, and the error is reported: `UNAVAILABLE: Internal: Failed to get object at s3:// trainingplatform/repo/common_business_model-base/1/model.graphdef due to exception: , error message: curlCode: 28, Timeout was reached.`
Can I increase the timeout？

**Triton Information**
What version of Triton are you using?
22.06

Are you using the Triton container or did you build it yourself?
Triton container

**To Reproduce**
Steps to reproduce the behavior.

Describe the models (framework, inputs, outputs), ideally include the model configuration file (if using an ensemble include the model configuration file for that as well).

**Expected behavior**

**Comment by rmccorm4** [comment]
Hi @lhy2749, thanks for filing this issue.

@kthui if this is coming from S3 SDK, does it expose any options for timeout that we can expose to users? If not, how about support for fetching large objects in chunks/ranges?

**Comment by kthui** [comment]
Yes, I think we can expose the timeout config. I wonder if S3 SDK does not internally fetch objects in chunks, added a ticket for the enchancement DLIS-4578.
